app/src/main/java/io/github/droidkaigi/confsched2017/di/OverlayViewComponent.java

```diff
@@ -0,0 +1,14 @@
+package io.github.droidkaigi.confsched2017.di;
+
+import dagger.Subcomponent;
+import io.github.droidkaigi.confsched2017.di.scope.OverlayViewScope;
+import io.github.droidkaigi.confsched2017.view.customview.DebugOverlayView;
+
+/**
+ * @author KeithYokoma
+ */
+@OverlayViewScope
+@Subcomponent(modules = {OverlayViewModule.class})
+public interface OverlayViewComponent {
```

Comment:
I think this is too much for this kind of small feature, but `Service` is too big and I would like to introduce the similar architecture on the overlay views to Fragments.